st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -106,6 +109,28 @@ def match(self, action_alias_match_api, **kwargs):
             pecan.abort(http_client.BAD_REQUEST, str(e))
             return [self._match_tuple_to_dict(match) for match in e.matches]
 
+    @request_user_has_permission(permission_type=PermissionType.ACTION_ALIAS_HELP)
```

Comment:
A bit late to the game, but perhaps we could / should just re-use existing list or get permission here.

In any case, not a big deal :)